build.gradle.kts

```diff
@@ -9,7 +9,7 @@ val rewriteVersion = rewriteRecipe.rewriteVersion.get()
 dependencies {
     implementation(platform("org.openrewrite:rewrite-bom:$rewriteVersion"))
     implementation("org.openrewrite:rewrite-java")
-    implementation("org.openrewrite.recipe:rewrite-java-dependencies:$rewriteVersion")
+    implementation("org.openrewrite.recipe:rewrite-spring:$rewriteVersion")
```

Comment:
I'm thinking we should actually invert this dependency: have rewrite-spring depend on rewrite-openapi.

The spring and springdoc related recipes can then move to rewrite-spring, such that this module only does swagger & openapi.
That would make it better usable for folks not necessarily using Spring, while allowing these recipes to be included there.